tools/operator/networkspec/structs.go

```diff
@@ -5,65 +5,39 @@
 package networkspec
 
 type Config struct {
-	ArtifactsLocation    string                 `yaml:"artifacts_location,omitempty"`
-	OrdererOrganizations []OrdererOrganizations `yaml:"orderer_organizations,omitempty"`
-	PeerOrganizations    []PeerOrganizations    `yaml:"peer_organizations,omitempty"`
+	ArtifactsLocation    string                 `yaml:"artifactsLocation,omitempty"`
+	OrdererOrganizations []OrdererOrganizations `yaml:"ordererOrganizations,omitempty"`
+	PeerOrganizations    []PeerOrganizations    `yaml:"peerOrganizations,omitempty"`
 	Orderer              struct {
 		OrdererType string `yaml:"orderertype,omitempty"`
 	} `yaml:"orderer,omitempty"`
-	NumChannels   int    `yaml:"num_channels,omitempty"`
+	NumChannels   int    `yaml:"numChannels,omitempty"`
 	TLS           string `yaml:"tls,omitempty"`
 	EnableNodeOUs bool   `yaml:"enableNodeOUs,omitempty"`
 	K8s           struct {
-		DataPersistence string `yaml:"data_persistence,omitempty"`
-		ServiceType     string `yaml:"service_type,omitempty"`
+		DataPersistence string `yaml:"dataPersistence,omitempty"`
+		ServiceType     string `yaml:"serviceType,omitempty"`
 	} `yaml:"k8s,omitempty"`
 }
 
 type OrdererOrganizations struct {
 	Name        string `yaml:"name,omitempty"`
-	MSPID       string `yaml:"msp_id,omitempty"`
-	NumOrderers int    `yaml:"num_orderers,omitempty"`
-	NumCA       int    `yaml:"num_ca,omitempty"`
+	MSPID       string `yaml:"mspId,omitempty"`
+	NumOrderers int    `yaml:"numOderers,omitempty"`
+	NumCA       int    `yaml:"numCa,omitempty"`
 }
 
 type KafkaConfig struct {
-	NumKafka             int `yaml:"num_kafka,omitempty"`
-	NumKafkaReplications int `yaml:"num_kafka_replications,omitempty"`
-	NumZookeepers        int `yaml:"num_zookeepers,omitempty"`
+	NumKafka             int `yaml:"numKafka,omitempty"`
+	NumKafkaReplications int `yaml:"numKafkaReplications,omitempty"`
+	NumZookeepers        int `yaml:"numZookeepers,omitempty"`
 }
 
 type PeerOrganizations struct {
 	Name     string `yaml:"name,omitempty"`
-	MSPID    string `yaml:"msp_id,omitempty"`
-	NumPeers int    `yaml:"num_peers,omitempty"`
-	NumCA    int    `yaml:"num_ca,omitempty"`
-}
-
-type MSP struct {
-	AdminPem   string `json:"admin_pem"`
-	CAPem      string `json:"ca_pem"`
-	TLSPem     string `json:"tls_pem"`
-	Pem        string `json:"pem"`
-	PrivateKey string `json:"private_key"`
-}
-
-type TLS struct {
-	CACert     string `json:"ca_cert"`
-	ServerCert string `json:"server_cert"`
-	ServerKey  string `json:"server_key"`
-}
-
-type CA struct {
-	Pem        string `json:"pem"`
-	PrivateKey string `json:"private_key"`
-}
-
-type Component struct {
-	Msp   MSP `json:"msp"`
-	TLS   TLS `json:"tls"`
-	CA    CA  `json:"ca"`
-	TLSCa CA  `json:"tlsca"`
```

Comment:
Did you intend to delete all these?

Reply:
yes.. We are not using them in the code

Reply:
We are not using them in the code @scottz64